clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetch.java

```diff
@@ -95,6 +102,13 @@ public int numRecords() {
         return numRecords;
     }
 
+    /**
+     * @return the next offsets and metadata (last epochs is included)
+     */
+    public Map<TopicPartition, OffsetAndMetadata> nextOffsetAndMetadata() {
+        return Collections.unmodifiableMap(nextOffsetAndMetadata);
```

Comment:
`Map.copyOf`

Reply:
`Map.of` itself returns an immutable map. Maybe no need to create a copy?